src/pages/Quiz/QuestionWrapper/index.js

```diff
@@ -68,7 +70,6 @@ const QuestionWrapper = (props) => {
     props.fetchSelectedQuestions(testAnswers.length);
     // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
why do you need to disable eslint here?

Reply:
fixed. update will be in next commit

Reply:
How have you fixed it?
If you can't fix it, tell me a reason.